model/accesstokenSchema.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const UserSignup = require('../model/signupschema')
+
```

Comment:
why do we need access token schema when we are using jwt

Reply:
because sir there is a question where I have to make schema for authentication where I don't use JWT thats why I am using accesstoken schema